conf/test_full.config

```diff
@@ -0,0 +1,19 @@
+/*
+ * -------------------------------------------------
+ *  Nextflow config file for running full-size tests
+ * -------------------------------------------------
+ * Defines bundled input files and everything required
+ * to run a full size pipeline test. Use as follows:
+ *   nextflow run nf-core/cageseq -profile test_full,<docker/singularity>
+ */
+
+params {
+  config_profile_name = 'Full test profile'
+  config_profile_description = 'Full test dataset to check pipeline function'
+
+  // Input data for full size test
+  input_paths = [
+    ['Testdata', ['https://github.com/nf-core/test-datasets/raw/exoseq/testdata/Testdata_R1.tiny.fastq.gz', 'https://github.com/nf-core/test-datasets/raw/exoseq/testdata/Testdata_R2.tiny.fastq.gz']],
+    ['SRR389222', ['https://github.com/nf-core/test-datasets/raw/methylseq/testdata/SRR389222_sub1.fastq.gz', 'https://github.com/nf-core/test-datasets/raw/methylseq/testdata/SRR389222_sub2.fastq.gz']]
+  ]
```

Comment:
Could copy the input data/parameters you are using for `test.config` in here for now.